sharedmodels/Pipfile

```diff
@@ -9,6 +9,10 @@ devtools = {editable = true,path = "./../devtools"}
 [packages]
 rdflib = "*"
 sharedmodels = {editable = true,path = "."}
+unidecode = "*"
```

Comment:
Did you run `pipenv-setup sync` after you made this change?

Reply:
I've introduced a new script called [pyright-cascase.sh](https://github.com/GSS-Cogs/csvwlib/pull/129/commits/be9e5afb2295c77530241076c275a292abb50cc5#diff-68a070bae6f3a3afdf3396f9a17a9e3b820b0f19f8a7f90ac77bf879e7b7cf6a) which cascades all package updates around the setup.py files as well as the pipfile.lock files